fix(authentication): sign-in with oauth provider failed (#538)
capawesome-team/capacitor-firebase: .changeset/fifty-eels-notice.md

---

> '@capacitor-firebase/authentication': minor

Reviewer: ```suggestion
'@capacitor-firebase/authentication': patch
```

---

> Fix swapped link and signIn functions for oauth flow sign in

Reviewer: ```suggestion
fix(ios): sign-in with oauth provider failed
```
